neon_hana/app/routers/util.py

```diff
@@ -34,11 +34,17 @@
 
 @util_route.get("/client_ip", response_class=PlainTextResponse)
 async def api_client_ip(request: Request) -> str:
-    client_manager.validate_auth("", request.client.host)
-    return request.client.host
+    ip_addr = request.client.host if request.client else "127.0.0.1"
+    if len(ip_addr.split('.')) != 4:
```

Comment:
nit(non-blocking): There is an outside chance a hostname could have multiple subdomains and end up triggering this conditional. Might be worth using regex, but this won't make or break the PR IMHO

Reply:
Found a rx string and added a check with some unit tests to better evaluate this. I forgot to go back through this; I didn't realize the host param could be a hostname until it came up in GHA tests